src/itinerary/index.tsx

```diff
@@ -47,58 +44,64 @@ const StyledItinerary = styled(Itinerary)`
   }
 
   & .kirk-itinerary-location time {
-    min-width: ${componentSizes.timeWidth};
+    min-width: calc(${gutterPaddingStart} + ${componentSizes.timeWidth});
   }
 
   & .kirk-itinerary-location-city {
     padding-left: ${space.l}; /* Adding the width of the step points to the regular spacing */
   }
 
-  &.kirk-itinerary--small .kirk-itinerary-location:not(.kirk-itinerary--arrival) div {
+  &.kirk-itinerary--small .kirk-itinerary-location:not(.kirk-itinerary--arrival)
+      .kirk-itinerary-location-city {
     padding-bottom: 0;
   }
 
-  & .kirk-itinerary-location div::before,
-  & .kirk-itinerary-location div::after {
+  & .kirk-itinerary-location .kirk-itinerary-location-city::before,
+  & .kirk-itinerary-location .kirk-itinerary-location-city::after {
     box-sizing: border-box;
     content: '';
     display: block;
     width: 4px;
     position: absolute;
   }
 
-  & .kirk-itinerary-location div::before {
+  & .kirk-itinerary-location .kirk-itinerary-location-city::before {
     height: 100%;
     background-color: ${color.primaryText};
     top: 0;
-    left: ${componentSizes.timeWidth};
+    left: calc(${gutterPaddingStart} + ${componentSizes.timeWidth});
   }
 
-  &.kirk-itinerary--small .kirk-itinerary-location div::before {
+  &.kirk-itinerary--small .kirk-itinerary-location .kirk-itinerary-location-city::before {
     left: 2px;
   }
 
-  & .kirk-itinerary-location div::after {
+  & .kirk-itinerary-location .kirk-itinerary-location-city::after {
     width: 8px;
     height: 8px;
     background-color: ${color.white};
     border: 2px solid ${color.primaryText};
     border-radius: 50%;
-    top: calc(4px + ${space.m});
-    left: calc(${componentSizes.timeWidth} - 2px);
+    top: calc(4px + ${space.m} + 4px);
+    left: calc(${gutterPaddingStart} + ${componentSizes.timeWidth} - 2px);
+  }
+  
+  & .kirk-itinerary-location.kirk-itinerary--departure .kirk-itinerary-location-city::after,
+  & .kirk-itinerary-location.kirk-itinerary--arrival .kirk-itinerary-location-city::after {
+    top: calc(4px + ${space.m} + 4px);
```

Comment:
the 4px corresponds to what ?

Reply:
It's just ugly adjustments to align stuff. I could use some magic constants to make it feel it was engineered if you prefer. We will need a much deeper refactoring if we want meaningful constants. For now, I will at least merge the 2 4px into one 8px :)